nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,296 @@
+
+If necessary, the application can retrieve the modem time counter value by calling the :c:func:`nrf_modem_dect_phy_time_get` function.
+
+Both radio state transitions between operations in the modem core and the inter-processor communication (IPC) mechanism between the application and more cores have latency.
+The latency of both radio state transitions and IPC is measured and provided by the DECT PHY interface upon initialization, by the :c:member:`nrf_modem_dect_phy_callbacks.init` callback function, in the :c:member:`nrf_modem_dect_phy_modem_cfg.latency` parameter.
+
+.. note::
+   The application must account for operations' latency when scheduling operations.
+
+Temperature monitoring
+======================
+
+The DECT PHY interface reports the current SoC temperature as measured by the DECT NR+ modem firmware in all radio operation callbacks, and upon initialization.
+This allows the application to track the changes in temperature and adjust further scheduling of operations accordingly, to ensure the device remains within safe operating temperatures.
+
+The DECT NR+ modem firmware has an internal temperature protection mechanism that prevents the SoC from operating above safe temperature limits.
+The operating temperature limit is reported upon initialization by the :c:member:`nrf_modem_dect_phy_callbacks.init` callback function, in the :c:member:`nrf_modem_dect_phy_modem_cfg.temperature_limit` parameter.
+
+If the temperature threshold is reached, the modem cancels all scheduled radio operations and reports the :c:enum:`nrf_modem_dect_phy_err.NRF_MODEM_DECT_PHY_ERR_OP_CANCELED` error in their relative callbacks.
+Additionally, it reject further scheduling of radio operations with the :c:enum:`nrf_modem_dect_phy_err.NRF_MODEM_DECT_PHY_ERR_TEMP_HIGH` error.
+
+In this event, the application must de-initialize the Modem library PHY interface (and thus the Modem firmware DECT NR+ PHY layer) by calling :c:func:`nrf_modem_dect_phy_deinit` and allow the device to cool.
+
+The application can then re-initialize the DECT PHY interface by calling :c:func:`nrf_modem_dect_phy_init`, and read the current measured temperature in the :c:member:`nrf_modem_dect_phy_callbacks.init` function parameters,
+to ensure the temperature has decreased below the allowed threshold.
+
+Physical layer capabilities
+===========================
+
+The application can retrieve the DECT NR+ modem firmware physical layer capabilities by calling the :c:func:`nrf_modem_dect_phy_capability_get` function.
+The list of supported capabilities is returned to the application in the :c:member:`nrf_modem_dect_phy_callbacks.capability_get` callback function.
+
+Radio operations
+****************
+
+The DECT PHY interface offers three radio operations: transmission (TX), reception (RX), and RSSI measurement.
+Each of these operations can be performed with different parameters.
+
```

Comment:
Should it be said in this high level  chapter that operations (RX,TX,RSSI, TX_RX) may be scheduled to start asap/immediately or with scheduled manner?

Reply:
ok

Reply:
actually this is mentioned already in the `modem time and operation latency` section above